homesensorhub/sensors/environmental_sensor.py

```diff
@@ -39,11 +40,26 @@ def collect_data(self) -> dict:
 
         :return: None
         """
+        sensor = self.get_sensor()
+
         sensor_data = {
-                     'temperature': self.get_sensor().temperature,
-                     'humidity': self.get_sensor().humidity,
-                     'pressure': self.get_sensor().pressure,
-                     'altitude': self.get_sensor().altitude
+            'temperature': self.build_sensor_packet('temperature',
+                                                    sensor.temperature,
+                                                    datetime.datetime.now(),
+                                                    'celsius'
+                                                    ),
+            'humidity': self.build_sensor_packet('humidity',
+                                                 sensor.humidity,
+                                                 datetime.datetime.now(),
+                                                 'percent'),
+            'pressure': self.build_sensor_packet('pressure',
+                                                 sensor.pressure,
+                                                 datetime.datetime.now(),
+                                                 'UF/USA'),
+            'altitude': self.build_sensor_packet('altitude',
+                                                 sensor.altitude,
+                                                 datetime.datetime.now(),
+                                                 'not sure'),
```

Comment:
Altitude is measured in meters. :)
Also, might be better to calculate the other way around: specify altitude as a setting for the sensor, and calculate pressure at sea-level, not just local pressure, to be able to compare with weather station values.

Reply:
I have created #9 to fix this. For now I will change the unit of measurement.

Reply:
I will include this fix in #9.